receiver/apachesparkreceiver/metadata.yaml

```diff
@@ -4,4 +4,689 @@ status:
+    description: Number of methods generated by the CodeGenerator.
+    enabled: true
+    sum:
+      aggregation: cumulative
+      monotonic: true
+      value_type: int
+    unit: "{ methods }"
+    attributes: []
+  spark.driver.code_generator.generated_method.average_size:
+    description: Average method size of the classes generated by the CodeGenerator.
+    enabled: true
+    gauge:
+      value_type: double
+    unit: bytes
+    attributes: []
+  spark.driver.code_generator.source_code.count:
+    description: Number of source code generation operations performed by the CodeGenerator.
+    enabled: true
+    sum:
+      aggregation: cumulative
+      monotonic: true
+      value_type: int
+    unit: "{ operations }"
+    attributes: []
+  spark.driver.code_generator.source_code.average_size:
+    description: Average size of the source code generated by a CodeGenerator code generation operation.
+    enabled: true
+    gauge:
+      value_type: double
+    unit: bytes
+    attributes: []
+  spark.driver.dag_scheduler.jobs.active:
+    description: Number of active jobs currently being processed by the DAGScheduler.
+    enabled: true
+    sum:
+      aggregation: cumulative
+      monotonic: false
+      value_type: int
+    unit: "{ jobs }"
+    attributes: []
+  spark.driver.dag_scheduler.jobs.all:
+    description: Number of jobs that have been submitted to the DAGScheduler.
+    enabled: true
+    sum:
+      aggregation: cumulative
+      monotonic: true
+      value_type: int
+    unit: "{ jobs }"
+    attributes: []
+  spark.driver.dag_scheduler.stages.failed:
+    description: Number of failed stages run by the DAGScheduler.
+    enabled: true
+    sum:
+      aggregation: cumulative
+      monotonic: true
+      value_type: int
+    unit: "{ stages }"
+    attributes: []
+  spark.driver.dag_scheduler.stages.running:
+    description: Number of stages the DAGScheduler is currently running.
+    enabled: true
+    sum:
+      aggregation: cumulative
+      monotonic: false
+      value_type: int
+    unit: "{ stages }"
+    attributes: []
+  spark.driver.dag_scheduler.stages.waiting:
+    description: Number of stages waiting to be run by the DAGScheduler.
+    enabled: true
+    sum:
+      aggregation: cumulative
+      monotonic: false
+      value_type: int
+    unit: "{ stages }"
+    attributes: []
+  spark.driver.live_listener_bus.events_posted:
+    description: Number of events that have been posted on the LiveListenerBus.
+    enabled: true
+    sum:
+      aggregation: cumulative
+      monotonic: true
+      value_type: int
+    unit: "{ events }"
+    attributes: []
+  spark.driver.live_listener_bus.listener_processing_time.average:
+    description: Average time taken for the LiveListenerBus to process an event posted to it.
+    enabled: true
+    gauge:
+      value_type: double
+    unit: ms
+    attributes: []
+  spark.driver.live_listener_bus.events_dropped:
```

Comment:
Combine with events_posted?

Reply:
My only concern would be that I don't think we could meaningfully add events_posted and events_dropped - since an event can be posted to the bus and then dropped. Thoughts?

Reply:
Ok, yeah, if these are not mutually exclusive, then no need to combine them.